modules/mono/editor/GodotTools/GodotTools/Build/BuildOutputView.cs

```diff
@@ -131,7 +131,7 @@ private void IssueActivated(int idx)
 
             BuildIssue issue = _issues[issueIndex];
 
-            if (string.IsNullOrEmpty(issue.ProjectFile) && string.IsNullOrEmpty(issue.File))
+            if (string.IsNullOrEmpty(issue.ProjectFile) || string.IsNullOrEmpty(issue.File))
```

Comment:
This would prevent showing warnings without an associated file in the MSBuild panel. 
And it's not needed after the changes to `eventSource_ErrorRaised` and `eventSource_WarningRaised`.

```suggestion
            if (string.IsNullOrEmpty(issue.ProjectFile) && string.IsNullOrEmpty(issue.File))
```